src/components/assistant_components/AssistantSettings.tsx

```
@@ -349,6 +350,19 @@ const AssistantSettings: React.FC<AssistantSettingsProps> = ({
                   <p className="text-xs text-gray-500 dark:text-gray-400 mt-1">Compatible with OpenAI API format</p>
                 </div>
               </button>
+              <button
+                onClick={() => handleApiTypeChange('litellm')}
+                className={`flex flex-col items-center p-4 rounded-lg border-2 transition-all ${
+                  apiType === 'litellm'
+                    ? 'border-sakura-500 bg-sakura-50 dark:bg-sakura-500/10'
+                    : 'border-gray-200 hover:border-sakura-200 dark:border-gray-700'
+                }`}
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)